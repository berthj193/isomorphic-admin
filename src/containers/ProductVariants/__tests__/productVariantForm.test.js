import React from 'react';
import ProductVariantForm from '../productVariantForm';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('<ProductVariantForm />', () => {
  const getComponent = (props = {}) => shallow(<ProductVariantForm {...props} />);

  it('should render properly', () => {
    expect(toJson(getComponent())).toMatchSnapshot();
  });
});
