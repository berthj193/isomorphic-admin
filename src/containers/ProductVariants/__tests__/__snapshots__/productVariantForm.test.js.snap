// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProductVariantForm /> should render properly 1`] = `
<Form
  hideRequiredMark={false}
  layout="horizontal"
  onSubmit={[Function]}
  prefixCls="ant-form"
>
  <Field
    component={[Function]}
    label="Default SKU"
    name="default_sku"
    required={true}
    validate={
      Array [
        [Function],
      ]
    }
  />
  <Field
    component={[Function]}
    name="default_options"
    required={true}
    validate={
      Array [
        [Function],
      ]
    }
  />
  <CustomComponent />
</Form>
`;
